pyTests/image/test_imageIO.py

```diff
@@ -17,8 +17,9 @@ def loop(image):
     optRead = img.ImageReadOptions(img.EImageColorSpace_NO_CONVERSION)
     optWrite = img.ImageWriteOptions()
     optWrite.toColorSpace(img.EImageColorSpace_NO_CONVERSION)
-    img.writeImage(new_path, image, optWrite)
-    
+    oiio_params = img.oiioParams(orientation, pixelAspectRatio, compression)
+    img.writeImage(new_path, image, optWrite, oiio_params.get())
```

Comment:
Remove the file at the end with try/finally like https://github.com/alicevision/AliceVision/blob/898b8fb17993687b4be94d9bc73eec2fa9a34eb2/pyTests/sfmDataIO/test_sfmdataio.py#L55